dummy-data-product/src/dependencies/cleaning/Remove_duplicate.py

```diff
@@ -0,0 +1,5 @@
+# Remove duplicate
+import pandas as pd
+def duplicate():

```

Comment:
There were plenty of things to be done in cleaning whereas only duplicates are being dropped here.